Configuration/AlCa/python/autoCond.py

```diff
@@ -35,19 +35,16 @@
     'run2_data_relval'  :   '113X_dataRun2_relval_v4',
     # GlobalTag for Run2 HI data
     'run2_data_promptlike_hi' : '113X_dataRun2_PromptLike_HI_v4',
-    # GlobalTag for Run1 HLT: it points to the online GT
-    'run1_hlt'          :   '101X_dataRun2_HLT_frozen_v11',
-    # GlobalTag for Run2 HLT: it points to the online GT
-    'run2_hlt'          :   '101X_dataRun2_HLT_frozen_v11',
-    # GlobalTag for Run2 HLT RelVals: customizations to run with fixed L1 Menu
-    'run2_hlt_relval'      :   '112X_dataRun2_HLT_relval_v4',
-    'run2_hlt_relval_hi'   :   '112X_dataRun2_HLT_relval_HI_v3',
-    # GlobalTag for Run2 HLT for HI (not 2018 HI): it points to the online GT
-    'run2_hlt_hi'       :   '101X_dataRun2_HLTHI_frozen_v11',
+    # GlobalTag for Run3 HLT: it points to the online GT
+    'run3_hlt'          :   '112X_dataRun3_HLT_v2',
+    # GlobalTag with fixed snapshot time for Run1 HLT RelVals: customizations to run with fixed L1 Menu
+    'run1_hlt_relval'   :   '112X_dataRun2_HLT_relval_v5',
+    # GlobalTag with fixed snapshot time for Run2 HLT RelVals: customizations to run with fixed L1 Menu
+    'run2_hlt_relval'   :   '112X_dataRun2_HLT_relval_v5',
     # GlobalTag for Run3 data relvals (express GT)
-    'run3_data_express'        :   '112X_dataRun3_Express_v3',
+    'run3_data_express'        :   '112X_dataRun3_Express_v4',
     # GlobalTag for Run3 data relvals
-    'run3_data_promptlike'     :   '112X_dataRun3_Prompt_v3',
+    'run3_data_promptlike'     :   '112X_dataRun3_Prompt_v4',
```

Comment:
Since this is a real infinite time snapshot GlobalTag suitable for Prompt Reco (with all the connected risks associated to it) what's the point of calling it 'promptlike'? Would not it better to call it `run3_prompt`? Also is the plan during  pp operations to keep updating it as frequently as Tier-0 does?

Reply:
I changed the key name to `run3_data_prompt` in commit f9b641c. Once an offline Run 3 GT is available, workflows that use `run3_data_prompt` would be converted to a `run3_data` key. Of course, it would be possible to create an offline GT for Run 3. But since that GT would currently have no other uses, I think it's preferable to exercise a GT that is actually used.

As long as there are no tags contained in the prompt GTs with synchronizations not allowed by the prompt queue, I think that the synchronization policy would be sufficient to ensure reproducibility. Is there another potential way to violate reproducibility that I am not considering?

Reply:
> As long as there are no tags contained in the prompt GTs with synchronizations not allowed by the prompt queue, I think that the synchronization policy would be sufficient to ensure reproducibility. Is there another potential way to violate reproducibility that I am not considering?

I am mostly concerned with private and "semi-official" (such as AlCaRelVals for conditions validation) usage of the `autoCond` keys. As soon as we have 24/7 operations someone might be tempted to use this key to process fresh data, possibly even earlier that PromptReco is released for that run. So in principle you might get different results with that recipe depending if you run before or after than the PCL and O2Os for that run have completed. For RelVals and unit tests using a fixed run (from the relatively distant past) the prompt synchronization will enforce reproducibility.

Reply:
With a fixed snapshot time, analysis of runs taken after the snapshot time could receive incorrect, old conditions. If someone tries to process data before the PromptReco is released for that run, they will run into problems independent of whether the prompt GT is used in autoCond or not. I will let the other AlCa conveners decide what they prefer to do here, as they will be maintaining these GTs in the future, not me.

Reply:
Maybe for the time being we could choose to keep this GT in autoCond as @christopheralanwest suggests and reassess it in the future, in case we think it is necessary.